sdk/servicebus/Azure.Messaging.ServiceBus/src/Sender/ServiceBusSender.cs

```diff
@@ -268,6 +269,24 @@ private DiagnosticScope CreateDiagnosticScope(IReadOnlyCollection<ServiceBusMess
                 DiagnosticScope.ActivityKind.Client);
 
             scope.SetMessageData(messages);
+
+            return scope;
+        }
+
+        private DiagnosticScope CreateDiagnosticScope(IReadOnlyCollection<AmqpMessage> messages, string activityName)
```

Comment:
nit: suggest taking in the ServiceBusMessageBatch here to help make it easier to grok why this method is needed as compared to the SBMessage one.